portfolio/src/main/webapp/forum.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>A Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+
+    <!------------------------------------------------------>
+    <!-------------------- Banner -------------------------->
+    <!------------------------------------------------------>
+    <div id="banner">
+      <h1>Anya's Forum</h1>
+      <h2>Its more like a basic commenting stuff</h2>
+      <hr>
+      <button onclick="document.location='./index.html'">Main</button> 
+      <button onclick="document.location='./gallery.html'">Gallery</button> 
+      <button onclick="document.location='./forum.html'">Forum</button> 
+      <!-- TODO: Going to put spacing, later--> 
+      <button>Toggle Mode</button>
```

Comment:
is this the light/dark theme you mentioned?

Reply:
Yeah, but it doesn't work yet.